strawberry/auto.py

```diff
@@ -24,11 +24,11 @@ class StrawberryAutoMeta(type):
 
     """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args: str, **kwargs: Dict[str, Any]):
```

Comment:
I think in all kwargs we want Any (or the correct type): https://mypy-play.net/?mypy=latest&python=3.11&gist=925cfe7dea572e1a3a0de141e8bd5288

Reply:
Will patch tonight. Thanks!

Reply:
> Will patch tonight. Thanks!

patched - but now seeing a single test error I'm not able to replicate locally...

stuck on this for now

Reply:
ok - not sure if it was a fluke or what - but I can't repro that previous test failure locally. Forced re-runs a couple of times be reverting and re-reverting. 🤷

Reply:
Increasingly looking like that one red build was a fluke. I'll try and keep this branch up to date for easier merging but let me know if anything else needs attention. Thanks @patrick91!